x/stake/keeper/migrate.go

```diff
@@ -27,3 +29,36 @@ func MigratePowerRankKey(ctx sdk.Context, keeper Keeper) {
 		keeper.SetNewValidatorByPowerIndex(ctx, val)
 	}
 }
+
+func MigrateValidators(ctx sdk.Context, k Keeper) {
+	store := ctx.KVStore(k.storeKey)
+	iterator := sdk.KVStorePrefixIterator(store, ValidatorsKey)
+	defer iterator.Close()
+
+	for ; iterator.Valid(); iterator.Next() {
+		validator := types.MustUnmarshalValidator(k.cdc, iterator.Value())
+		validator.DistributionAddr = types.GenerateDistributionAddr(validator.OperatorAddr, types.MockSideChainIDForBeaconChain)
+		k.SetValidator(ctx, validator)
+		delegation, found := k.GetDelegation(ctx, validator.FeeAddr, validator.OperatorAddr)
+		if !found {
+			panic(fmt.Sprintf("self delegation for %s not found", validator.OperatorAddr))
+		}
+		delegation.Height = ctx.BlockHeight()
+		k.SetDelegation(ctx, delegation)
+	}
+}
+
+func MigrateWhiteLabelOracleRelayer(ctx sdk.Context, k Keeper) {
```

Comment:
don't find where this function is called

Reply:
it's called in `node` repo, https://github.com/bnb-chain/node/pull/880/files#diff-0f1d2976054440336a576d47a44a37b80cdf6701dd9113012bce0e3c425819b7R581